### PR TITLE
Add building tutorial for POSIX systems

### PR DESCRIPTION
**Pull request by VictorLamoine**
Intents to replace http://www.pointclouds.org/downloads/source.html
The links at http://www.pointclouds.org/downloads/ should be updated (don't forget #839 as well)

The tutorial is nearly a copy, I changed the version number and the formatting is not always the same.
There is no logo for `POSIX` so I kept the default PCL logo. Using a Tux logo is not a good idea because the tutorial also targets MacOS X users etc.

### CODE REVIEW

**Comment by VictorLamoine** [comment]
Looks ok?

File: doc/tutorials/content/posix_building_pcl.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+`our tutorials page <http://www.pointclouds.org/documentation/tutorials/index.php>`_.
+
+.. contents::
+
+Stable
+======
+
+For systems for which we do not offer precompiled binaries, you need to compile Point Cloud Library (PCL) from source. Here are the steps that you need to take:
+Go to `Github <https://github.com/PointCloudLibrary/pcl/releases>`_ and download the version number of your choice.
+Uncompress the tar-bzip archive, e.g. (replace 1.6.0 with the correct version number)::
```
Comment (taketwo) [inline doc/tutorials/content/posix_building_pcl.rst]: 1.7.2

File: doc/tutorials/content/posix_building_pcl.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+  tar xvfj pcl-pcl-1.7.2.tar.gz
```
Comment (taketwo) [inline doc/tutorials/content/posix_building_pcl.rst]: `pcl-` repeated twice

File: doc/tutorials/content/posix_building_pcl.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Run the CMake build system using the default options::
+
+  cmake ..
+
+Or change them  (uses cmake-curses-gui)::
+
+  ccmake ..
+
+Please note that cmake might default to a debug build. If you want to compile a release build of PCL with enhanced compiler optimizations, you can change the build target to "Release" with "-    DCMAKE_BUILD_TYPE=Release".
```
Comment (taketwo) [inline doc/tutorials/content/posix_building_pcl.rst]: extra spaces in "-     DCMAKE_BUILD_TYPE=Release"

File: doc/tutorials/content/posix_building_pcl.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+  cmake -DCMAKE_BUILD_TYPE=Release .. 
+
+Finally compile everything (see `compiler_optimizations <http://www.pointclouds.org/documentation/advanced/compiler_optimizations.php>`_)::
+
+  make- j2
```
Comment (taketwo) [inline doc/tutorials/content/posix_building_pcl.rst]: minus should be in front of `j`

File: doc/tutorials/content/posix_building_pcl.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+or alternatively, if you did not change the  variable which declares where PCL should be installed, do::
+
+  sudo make install
+
+Here's everything again, in case you want to copy & paste it::
+
+  cd pcl-pcl-1.7.2 && mkdir build && cd build
+  cmake -DCMAKE_BUILD_TYPE=Release .. 
+  make
```
Comment (taketwo) [inline doc/tutorials/content/posix_building_pcl.rst]: does not match previously suggested `make -j2`

File: doc/tutorials/content/posix_building_pcl.rst [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+If you are eager to try out a certain feature of PCL that is currently under development (or you plan on developing and contributing to PCL), we recommend you try checking out our source repository, as shown below. If you're just interested in browsing our source code, you can do so by visiting `https://github.com/PointCloudLibrary/pcl <https://github.com/PointCloudLibrary/pcl>`_.
+
+Clone the repository::
+
+  git clone https://github.com/PointCloudLibrary/pcl pcl-trunk
+
+Please note that above steps (3-5) are almost identical for compiling the experimental PCL trunk code::
+
+  cd pcl-trunk && mkdir build && cd build
+  cmake -DCMAKE_BUILD_TYPE=Release ..
```
Comment (taketwo) [inline doc/tutorials/content/posix_building_pcl.rst]: I would put `RelWithDebInfo` here as it is more suited for development.

File: doc/tutorials/content/posix_building_pcl.rst (hunk 2 above)
Comment (VictorLamoine) [inline doc/tutorials/content/posix_building_pcl.rst]: It's the name of the archive when you download it from github; try it [here](https://github.com/PointCloudLibrary/pcl/releases/tag/pcl-1.7.2)

**Comment by taketwo** [comment]
Hmm, indeed!

**Comment by soyersoyer** [comment]
Boost 1.54? Not 1.46?

**Comment by VictorLamoine** [comment]
If building with OpenNI boost minimum version is 1.47 (first version with chrono)
Without OpenNI boost minimum version is 1.40

See #276

**Comment by VictorLamoine** [comment]
@taketwo Something must be wrong with the tutorial auto-update because the tutorial is not yet online.
I can generate the tutorial on my side, it looks ok.

**Comment by VictorLamoine** [comment]
Friendly ping! :ghost:

**Comment by taketwo** [comment]
Should be ok now. BTW, it occurred to me that it might make sense to rename it into "Compiling PCL from source on POSIX compliant systems" so that it stands in line with tutorials for the other systems. (The source file also, "compiling_pcl_posix.rst".)

**Comment by VictorLamoine** [comment]
Yes it's online thanks
Ok I'll do that with the other fixes (boost version, maybe dead links)

**Comment by taketwo** [comment]
:+1:

**Pull request merged**